lower/llpcSpirvLowerImageOp.cpp

```diff
@@ -600,6 +810,93 @@ void SpirvLowerImageOp::visitCallInst(
     }
 }
 
+// =====================================================================================================================
+// Tracks descriptor loading to handle CFG.
+llvm::Value* SpirvLowerImageOp::TrackDescriptorLoad(
+    Value*                   pOperand,                // Value to track from
+    DescriptorType           descType,                // Descriptor type
+    ShaderImageCallMetadata* pImageCallMeta,          // [out] Image call metadata
+    ConstantInt**            ppMemoryQualifier)       // [out] Memory qualifier
+{
+    Value*       pDescriptor = nullptr;
+
+    if (isa<SelectInst>(pOperand))
```

Comment:
I fail to find how phi instruction is handled. could you help to point it out? thanks!

Reply:
Phi instruction is not handled in this change, I didn't find a phi case yet, I think we can address this in next change.

Reply:
In my case phi is transformed to select during prior passes.